api_app/analyzers_manager/models.py

```diff
@@ -30,8 +31,9 @@ class AnalyzerReport(models.Model):
     errors = postgres_fields.ArrayField(
         models.CharField(max_length=512, blank=True, default=list)
     )
-    start_time = models.DateTimeField()
-    end_time = models.DateTimeField()
+    start_time = models.DateTimeField(default=timezone.now)
+    process_time = models.FloatField(default=0)
```

Comment:
we don't need `process_time` as it can be calculated dynamically from `start_time` and `end_time`

Reply:
^ yep. And is now a `@property` field in the `AbstractReport` model.